src/SaaS.SDK.Client.DataAccess/Context/SaasKitContext.cs

```diff
@@ -45,7 +47,8 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
         {
             if (!optionsBuilder.IsConfigured)
             {
-
+#warning To protect potentially sensitive information in your connection string, you should move it out of source code. See http://go.microsoft.com/fwlink/?LinkId=723263 for guidance on storing connection strings.
+                optionsBuilder.UseSqlServer("Server=contosodemosqlsrv.database.windows.net;Database=AMPSaaSDB1;User Id=saasadmin@contosodemosqlsrv.database.windows.net;Password=Hello789#;");
```

Comment:
Remove the test connection string from the code.

Reply:
sensitive information removed